### PR TITLE
Rename kpm commands

### PR DESCRIPTION
**Pull request by ChengTian**
parent #967 

Follow-up reaction: #1035

Impacted units: Tooling, test scripts, build scripts, wiki pages

### CODE REVIEW

**Comment by Praburaj** [comment]
Just a note : https://github.com/aspnet/MusicStore/blob/dev/test/E2ETests/Common/DeploymentUtility.cs#L313 needs to change as well with this change.

File: src/Microsoft.Framework.PackageManager/Program.cs [hunk 1]
```diff
@@ -142,9 +142,9 @@ public int Main(string[] args)
                 });
             });
 
-            app.Command("build", c =>
+            app.Command("pack", c =>
             {
-                c.Description = "Build NuGet packages for the project in given directory";
+                c.Description = "Pack NuGet packages for the project in given directory";
```
Comment (davidfowl) [inline src/Microsoft.Framework.PackageManager/Program.cs]: Build is still a better verb to use.

**Comment by ChengTian** [comment]
Changed script names in the latest commit:
packExclude -> bundleExclude
prepack -> prebundle
postpack -> postbundle
prebuild -> prepack
postbuild -> postpack

We also have `prebuild` and `postbuild` that serves for assembly-only scenarios.

File: src/Microsoft.Framework.PackageManager/Program.cs [hunk 2]
```diff
@@ -142,7 +142,7 @@ public int Main(string[] args)
                 });
             });
 
-            app.Command("build", c =>
+            app.Command("pack", c =>
             {
                 c.Description = "Build NuGet packages for the project in given directory";
```
Comment (dougbu) [inline src/Microsoft.Framework.PackageManager/Program.cs]: ? "Produce NuGet packages..."

**Comment by dougbu** [comment]
:shipit: from my perspective but I did not check thoroughly enough to be the primary reviewer.

will this lead to additional changes in the tooling schema?  other (new) downstream changes?

**Comment by ChengTian** [comment]
Need a rebase after https://github.com/aspnet/XRE/pull/1117 is checked in.

**Comment by ChengTian** [comment]
We will do this in release branch. Please check the new PR: #1124

**Pull request closed without merging**